### DISCUSSION TITLE
Get Starstruck (Help)

### DISCUSSION THREAD

**Discussion by faintout** [Schweinepriester/github-profile-achievements#187] [Q&A] [answered]
Hi all,
for get starstruck need to reach 16 star. If u want u can put star on my readme repository, I will do the same for you.
https://github.com/faintout/ThreeRoomDemo

**Comment by Luscarvalho** [comment, marked as answer]
Return the favor ;)

https://github.com/Luscarvalho/Luscarvalho

**Reply by 404jv** [reply]
Done. Can you star this https://github.com/404jv/dev-landia

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by ArielMAJ** [reply]
@404jv, @bacca87  I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by znetworkx** [reply]
hi

**Reply by orzklv** [reply]
Done!
https://github.com/katsuki-yuri/neko

**Reply by Rantoryu** [reply]
@Luscarvalho @number0x0001 @bacca87 @katsuki-yuri I've starred your repos.
I would appreciate your star too :)
https://github.com/Rantoryu/Personal_Website

**Comment by 404jv** [comment]
Done. Please leave one start here https://github.com/404jv/dev-landia

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by orzklv** [reply]
Done!
https://github.com/katsuki-yuri/neko

**Comment by Crushoverride007** [comment]
done. join the troops guys https://github.com/Crushoverride007/Crushoverride007

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by orzklv** [reply]
Done! https://github.com/katsuki-yuri/neko

**Comment by nel0x** [comment]
Done. Please also leave a star at mine: https://github.com/nel0x/arch-installer

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by nguyen-tuan-hiep** [reply]
Done! Can you give me a star on my repo: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering

**Reply by ArielMAJ** [reply]
@nel0x I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by faintout** [reply]
start！please reply

**Comment by bacca87** [comment]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by kshitij-halankar** [reply]
@bacca87 Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by ArielMAJ** [reply]
@bacca87 I starred you a while back. Removing your star until you starback: https://github.com/ArielMAJ/test.me
@kshitij-halankar feel free to star me too and I'll star you back.

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Reply by ArielMAJ** [reply]
@danielr1996  I starred you a while back. Removing your star until you starback: https://github.com/ArielMAJ/test.me

**Comment by kshitij-halankar** [comment]
@faintout Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by HHNebula** [comment]
Done!
https://github.com/HHNebula/HHNebula

**Reply by ArielMAJ** [reply]
I still haven't received your star, so I'm removing the one i gave you. Star me and I'll star you back: https://github.com/ArielMAJ/test.me

**Comment by bm4578** [comment]
Have finished ,Can you help me? https://github.com/bm4578/docker.git

**Reply by Rantoryu** [reply]
@bm4578 I've starred your repo.
Please star mine too :)
https://github.com/Rantoryu/Personal_Website

**Comment by 1wos** [comment]
I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Comment by orzklv** [comment]
swap time :D 
https://github.com/katsuki-yuri/neko

**Reply by ArielMAJ** [reply]
@katsuki-yuri I starred you, you can check at https://github.com/katsuki-yuri/neko/stargazers

Please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)

**Reply by Rantoryu** [reply]
@YuZhangWang if You can star my repo too I would appreciate it. I gave You a star
https://github.com/Rantoryu/Personal_Website

**Comment by Rantoryu** [comment]
@faintout 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Reply by 1wos** [reply]
I marked your repo with stars! Can you do it for me, too?
https://github.com/sommizzu/Big-Data-Convergence

**Reply by Rantoryu** [reply]
@sommizzu already did :) thank You for starring mine.

**Comment by Rantoryu** [comment]
@faintout I gave You a star.
https://github.com/Rantoryu/Personal_Website
